client/src/containers/NychaPage.tsx

```diff
@@ -312,8 +330,9 @@ class NychaPageWithoutI18n extends Component {
                 <SocialShare
                   location="nycha-page"
                   url={
-                    usersInputAddress &&
-                    `${getSiteOrigin()}${createRouteForAddressPage(usersInputAddress)}`
+                    usersInputAddress
+                      ? `${getSiteOrigin()}${createRouteForAddressPage(usersInputAddress)}`
+                      : undefined
```

Comment:
Also out-of-scope of this PR, but it seems like both the generated URL and the ternary here are the same as in the not registered page, which makes me wonder if it'd be worth refactoring something to be more DRY... Not a huge deal though.

Reply:
Sweet! Will address this in a resulting PR